### ISSUE TITLE
Issue with the atom labels

### ISSUE THREAD

**Issue by MaximeScope** [lab-cosmo/chemiscope#300]
When a user loads the "Chemical Shieldings" example and attempts to display atom labels on any 3dmol screen (by enabling the "atom labels" option in the "Visualizer settings"), the labels seem misaligned.

**Comment by alexpalade** [comment]
Can confirm, here's a [screenshot](https://github.com/lab-cosmo/chemiscope/assets/1126946/6adc03a0-2db8-4dec-8c8f-c3e2cdfd300d). The example loads the `/app/examples/CSD-1000R.json.gz` file.

**Comment by ceriottm** [comment]
Yes, I know what's happening. The labels are shown for all atoms in the unit cell, but are not replicated, nor subselected when the environment is highlighted.
